Cargo.toml

```diff
@@ -46,7 +46,7 @@ futures = "0.3.30"
 serde_cbor = "0.11.2"
 log = "0.4.22"
 itertools = "0.13.0"
-
```

Comment:
nit: Remove this

Reply:
@ratankaliani Remove what?

Reply:
The `tiny-keccak` patch. It's included at the bottom of the `Cargo.toml` as well.